src/javaRosa.js

```diff
@@ -1613,6 +1694,13 @@ define([
                             xmlWriter.writeXML(xml.normalize(val));
                             xmlWriter.writeEndElement();
                         }
+                        if (writeMarkdown) {
```

Comment:
Would it work to change this to `if (item.hasMarkdown)` and eliminate the local variable?